automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimePatchSpec.scala

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package runtimes
+
+import org.broadinstitute.dsde.workbench.google2.MachineTypeName
+import org.scalatest.time.{Minutes, Seconds, Span}
+import org.scalatest.DoNotDiscover
+import org.broadinstitute.dsde.workbench.service.util.Tags
+
+@DoNotDiscover
+class RuntimePatchSpec extends RuntimeFixtureSpec with LeonardoTestUtils {
+
+  //this is an end to end test of the pub/sub infrastructure
```

Comment:
minor, add a TODO to reenable this when bug is fixed

Reply:
Added